appveyor.yml

```diff
@@ -73,6 +76,9 @@ environment:
     #- TOOLCHAIN: "vs-14-2015"
     #  PROJECT_DIR: examples\LAPACK-CBLAS
 
+    #- TOOLCHAIN: "vs-14-2015-win64"
```

Comment:
I don't see this toolchain in original file. To remove.

Reply:
Should I add it in a separate PR? I'd like to test `vs-14-2015 win64` for the whole ceres-toolchain

Reply:
OH, it's the CBLAS test, sorry, will remove